pkg/features/kube_features.go

```diff
@@ -198,7 +198,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	ExperimentalHostUserNamespaceDefaultingGate: {Default: false, PreRelease: utilfeature.Beta},
 	ExperimentalCriticalPodAnnotation:           {Default: false, PreRelease: utilfeature.Alpha},
 	Accelerators:                                {Default: false, PreRelease: utilfeature.Alpha},
-	DevicePlugins:                               {Default: false, PreRelease: utilfeature.Alpha},
+	DevicePlugins:                               {Default: true, PreRelease: utilfeature.Alpha},
```

Comment:
Why are we auto-enabling an Alpha?

Reply:
i question the motivation of this PR.

simplify device plugin deployment logic doesn't make sense.  either device plugins are beta (and enabled) or alpha (and disabled with opt-in).  i don't really follow how deployment of a device plugin is the mechanism for opt-in.

Reply:
I think the main argument is that we expect the feature to take no effect if no device plugins are deployed in the cluster, although I agree that all of Alpha features seem to have been disabled by default.

Reply:
but by definition I would expect install of a device plugin on clusters to not work by default precisely because it’s not enabled by default.  If we want to promote device plugins to beta and enable by default, let’s discuss that and weigh the risks of doing that at this time.